src/app/utils.py

```diff
@@ -432,17 +581,18 @@ def get_license_data(issues):
     licenseIds = []
     licenseTexts = []
     for issue in issues:
-        if issue.get('pull_request') is None:
-            licenseInfo = issue.get('body')
-            if '[SPDX-Online-Tools]' in issue.get('title'):
-                licenseIdentifier = re.search(r'(?im)short identifier:\s([a-zA-Z0-9|.|-]+)', licenseInfo).group(1)
-                licenseIds.append(licenseIdentifier)
-                try:
-                    licenseXml = str(LicenseRequest.objects.get(shortIdentifier=licenseIdentifier).xml)
-                    licenseText = parseXmlString(licenseXml)['text']
-                    licenseTexts.append(clean(licenseText))
-                except LicenseRequest.DoesNotExist:
-                    pass
+        if not isinstance(issue, unicode):
```

Comment:
Same comment as above. unicode is not a builtin anymore on Python3. But the other issue lurking is why you would test that you not a string and then assume this is a mapping below? You would be 100% better off to make a positive explicit check that you have a mapping.

Reply:
Here this was a conflict with @Ugtan 's commit. Given that I do not have a clear undersanding of his project, I thought it wise to do some verifications.

Reply:
@tjasmith Don't get me wrong but if you had no idea about the code you should have made modifications in the first place or at least have pinged me, I would have definitely replied.